packages/liveblocks-node/src/utils.ts

```diff
@@ -94,3 +94,26 @@ export function urljoin(
   }
   return url.toString();
 }
+
+declare const brand: unique symbol;
+
+type Brand<T, TBrand extends string> = T & { [brand]: TBrand };
```

Comment:
The `Brand` util is exported in `@liveblocks/core` since recently, so feel free to DRY that up!